shared/common-adapters/avatar.desktop.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import React, {Component} from 'react'
+import resolveRoot from '../../desktop/resolve-root'
+import type {Props} from './avatar'
+
+const noAvatar = `file:///${resolveRoot('shared/images/no-avatar@2x.png')}`
+
+export default class Avatar extends Component {
```

Comment:
https://keybase.atlassian.net/browse/DESKTOP-541 to port other avatars to use this